packages/apollo-server-hapi/src/ApolloServer.ts

```diff
@@ -138,6 +111,7 @@ export class ApolloServer extends ApolloServerBase {
             : {
                 cors: cors !== undefined ? cors : true,
               },
+        uploadsConfig: this.uploadsConfig,
```

Comment:
Pass the upload configuration to the plugin for use within the route handler (see comments below).